src/consts/events-consts.ts

```diff
@@ -0,0 +1,2 @@
+export const PLAYGROUND_STORY_PREPARED = "playground-story-prepared";
```

Comment:
What does it stand for? Like "ready"?

Reply:
Yep, left same naming convention of Sgorybook